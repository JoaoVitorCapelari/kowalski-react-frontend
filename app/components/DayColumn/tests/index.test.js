import React from 'react';
import { mount } from 'enzyme';
import { IntlProvider, injectIntl } from 'react-intl';
import { fromJS } from 'immutable';

import { DayColumn } from '../index';
import { translationMessages } from '../../../i18n';

describe('<DayColumn />', () => {
  it('renders', () => {
    const renderedComponent = mount(
      <IntlProvider locale="en" messages={translationMessages.en}>
        {
          React.createElement(injectIntl(DayColumn), {
            day: new Date(2014, 1, 1),
            onFreeSlotClick: jest.fn(),
            onSlotClicked: jest.fn(),
            timeRecords: fromJS([]),
          })
        }
      </IntlProvider>
    );
    expect(renderedComponent.html()).toMatchSnapshot();
  });
});
