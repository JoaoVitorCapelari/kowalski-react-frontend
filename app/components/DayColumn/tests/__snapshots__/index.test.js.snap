// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DayColumn /> renders 1`] = `"<div class=\\"DayColumn__Wrapper-giFadl feTokT\\"><div class=\\"DayColumn__DayLabelWrapper-jaSPPb cLwSvO\\"><p>Sat</p><p>1</p></div><div class=\\"DayColumn__SlotsContainer-cgmRlA eZlanx\\"><a class=\\"sc-bdVaJa dVJLzw\\" role=\\"button\\" tabindex=\\"0\\">+</a></div></div>"`;
